CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 1.1.0 - 21 Apr 2020
+### Added
```

Comment:
Docker installer support?

Reply:
Very few people will ever encounter this, and it's not a feature users will get excited about.